src/fides/api/ops/api/v1/endpoints/privacy_experience_config_endpoints.py

```diff
@@ -0,0 +1,327 @@
+
+    # Fail early if the ExperienceConfig is going to be incompatible with any of the regions
+    validate_experience_config_and_region_compatibility(
+        db=db,
+        component_type=experience_config_data.component,
+        delivery_mechanism=experience_config_data.delivery_mechanism,
+        regions=new_regions,
+    )
+
+    logger.info(
+        "Creating experience config of component '{}' and delivery mechanism '{}'.",
+        experience_config_data.component.value,
+        experience_config_data.delivery_mechanism.value,
+    )
+    experience_config = PrivacyExperienceConfig.create(
+        db, data=experience_config_dict, check_name=False
+    )
+
+    if new_regions:
+        logger.info(
+            "Linking regions: {} to experience config '{}'",
+            human_friendly_list([reg.value for reg in new_regions]),
+            experience_config.id,
+        )
+    linked, unlinked, skipped = upsert_privacy_experiences_after_config_update(
+        db, experience_config, new_regions
+    )
+
+    return ExperienceConfigCreateOrUpdateResponse(
+        experience_config=experience_config,
+        linked_regions=linked,
+        unlinked_regions=unlinked,
+        skipped_regions=skipped,
+    )
+
+
+@router.get(
+    urls.EXPERIENCE_CONFIG_DETAIL,
+    status_code=HTTP_200_OK,
+    response_model=ExperienceConfigResponse,
+    dependencies=[
+        Security(verify_oauth_client, scopes=[scope_registry.PRIVACY_EXPERIENCE_READ])
+    ],
+)
+def experience_config_detail(
+    *,
+    db: Session = Depends(deps.get_db),
+    experience_config_id: str,
+) -> PrivacyExperienceConfig:
+    """
+    Returns a PrivacyExperienceConfig.
+    """
+    logger.info("Retrieving experience config with id '{}'.", experience_config_id)
+    return get_experience_config_or_error(db, experience_config_id)
+
+
+@router.patch(
+    urls.EXPERIENCE_CONFIG_DETAIL,
+    status_code=HTTP_200_OK,
+    response_model=ExperienceConfigCreateOrUpdateResponse,
+    dependencies=[
+        Security(
+            verify_oauth_client,
+            scopes=[
+                PRIVACY_EXPERIENCE_UPDATE,
+            ],
+        )
+    ],
+)
+def experience_config_update(
+    *,
+    db: Session = Depends(deps.get_db),
+    experience_config_id: str,
+    experience_config_data: ExperienceConfigUpdate,
+) -> ExperienceConfigCreateOrUpdateResponse:
+    """
+    Update Experience Config and then attempt to upsert Experiences and link back to ExperienceConfig.
+
+    All regions that should be linked to this ExperienceConfig (or remain linked) need to be
+    included in this request.
+    """
+    experience_config: PrivacyExperienceConfig = get_experience_config_or_error(
+        db, experience_config_id
+    )
+    experience_config_data_dict: Dict = experience_config_data.dict(exclude_unset=True)
+    # Pop the regions off the request
+    regions: List[PrivacyNoticeRegion] = experience_config_data_dict.pop("regions")
```

Comment:
You can update most fields individually for a PrivacyExperienceConfig, but one field that I have that needs to be included each time are the regions.  This is because I want to make it possible to unlink all regions too.  If no regions are supplied in the request, all PrivacyExperiences that were previously linked will be unlinked.

Another reason I think regions should be included here is this field in the request body actually pertains to resources in a different table. There are two operations here.  Update the ExperienceConfig, and then Upsert the Privacy Experiences.